drivers/rpmsg/rpmsg_syslog.c

```diff
@@ -0,0 +1,157 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Copyright (C) 2017 Pinecone Inc.
```

Comment:
Some description here for what services this file provides.

Reply:
Ok, I will add in the next pull.